wp-content/themes/core/assets/js/admin.js

```diff
@@ -1 +1,12 @@
+/**
+ * unregister some block styles we don't need
+ */
+
+wp.domReady( () => {
```

Comment:
@dpellenwood we haven't talked much about the JS directory's architecture. I reviewed SQ1 and felt like it was a lot that we potentially don't need. I figured more discussion and a separate PR could be made for building it out, which is why this is implemented in a super basic way here.